.gatherpress-org/playground-preview/plugin-proxy.php

```diff
@@ -0,0 +1,322 @@
+<?php
+/**
+ * https://gatherpress.org/some-random-chars/plugin-proxy.php?org=GatherPress&repo=gatherpress&workflow=Build%20GatherPress%20Plugin%20Zip&artifact=gatherpress-pr&pr=666
+ * 
+ * 
+ * Based on: https://github.com/WordPress/wordpress-playground/blob/ca759ee3281837596bfdd9736bbea205104741b2/packages/playground/website/public/plugin-proxy.php
+ */
+ini_set( 'display_errors', 0 );
+
+putenv( 'GITHUB_TOKEN=___REPLACE_WITH_REAL_GITHUB_TOKEN___' );
```

Comment:
Replace `___REPLACE_WITH_REAL_GITHUB_TOKEN___` with a working PAT (to comment on PRs).

Only in PRODUCTION on gatherpress.org, NOT here in the repo!